android/app/build.gradle

```diff
@@ -101,8 +101,8 @@ android {
         applicationId "com.agritrader"
         minSdkVersion 16
         targetSdkVersion 22
-        versionCode 3
-        versionName "1.2-alpha"
+        versionCode 6
+        versionName "1.6-alpha"
```

Comment:
So it's not `beta`? Everyone keeps calling it differently

Reply:
I'm deploying it on the appstore as alpha, that's why I called it alpha.

But our message in the app says "beta" which is fine, when we actually deploy to africa, it will be beta.